styleguide/index.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
```

Comment:
uhm... are we using tabs instead of spaces here? 🙅

Reply:
:S